java/src/org/openqa/selenium/chromium/ChromiumNetworkConditions.java

```diff
@@ -0,0 +1,107 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.chromium;
+
+import java.time.Duration;
+
+/**
+ * Provides manipulation of getting and setting network conditions from Chromium.
+ *
+ */
+public class ChromiumNetworkConditions {
+  private Boolean offline = false;
+  private Duration latency = Duration.ZERO;
+  private Number downloadThroughput = -1;
+  private Number uploadThroughput = -1;
+
+  public static final String OFFLINE = "offline";
+  public static final String LATENCY = "latency";
+  public static final String DOWNLOAD_THROUGHPUT = "download_throughput";
+  public static final String UPLOAD_THROUGHPUT = "upload_throughput";
+
+  /**
+   *
+   * @return whether network is simulated to be offline.
+   */
+  public Boolean getOffline() {
```

Comment:
`boolean` not `Boolean`. People are going to use this and expect boxing to work, which implies they'll never think of the `null` value.